decidim-core/app/views/layouts/decidim/header/_main_links_mobile_account.html.erb

```diff
@@ -20,7 +25,9 @@
               ) %>
       </div>
     <% else %>
-      <%= icon "user-smile-line", class: "main-bar__avatar" %>
+      <div class="main-bar__avatar-name">
+        <%= text_initials(current_user.name) %>
```

Comment:
Related to my other comment about this new pattern, I think we should implement it for desktop in this same PR. Can you do so @ElviaBth 🙏🏽 ? 

I'm talking about this "Account" icon:

![Screenshot of the desktop header](https://github.com/decidim/decidim/assets/717367/b6e0cd98-38a7-4d87-a693-92b9034c122b)

Reply:
Always happy to help.